examples/display_section.py

```diff
@@ -1,8 +1,9 @@
 import xarray as xr
 import xrviz
 from xrviz.display import Display
+from xrviz.sample_data import great_lakes
```

Comment:
Same as above.
In fact, I'm not sure you need this module at all any more.
I would prefer that you implement an example module within the xrviz package, such that
```
import xrviz; xrviz.example()
```
runs the full interface, but that the example data is *not* loaded unless either this is called, or it is ecxplicitly imported.

Reply:
I have deleted `control_section` and `display_section` since they are no longer required. `import xrviz; xrviz.example()`  will be used.